cfgov/research_reports/jinja2/index.html

```diff
@@ -0,0 +1,114 @@
+{% set is_report = True %}
+{% extends 'layout-side-nav.html' %}
+
+{% import './pdf-download.html' as pdf_download with context %}
+{% import 'templates/streamfield-sidefoot.html' as streamfield_sidefoot with context %}
+
+{% block css %}
+{{ super() }}
+<link rel="stylesheet" href="{{ static('apps/research-reports/css/main.css') }}">
+{% endblock %}
+
+{% block content_main_modifiers -%}
+    {{ super() }} content__flush-bottom research-report
+{%- endblock %}
+
+{% if report_sections is not defined  %}
+    {% set report_sections = get_report_sections(request, page) %}
+{% endif %}
+
+{% if report_appendices is not defined  %}
+    {% set report_appendices = get_report_appendices(request, page) %}
+{% endif %}
+
+{% set researchers = get_researchers() %}
+
+
+{% macro report_part(sections, is_appendix) %}
+  {% for section in sections -%}
+    <h2 class="report-header">
+      <a
+        id="{{section.url[1:]}}"
+        href="{{section.url}}">
+          {{('Appendix ' if is_appendix else '') + section.numbering + section.title}}
```

Comment:
Nitpicky thing, but @anselmbradford has asked me many times to use more spacing in Jinja templates. This is just one example from this PR, and feel free to ignore, especially since I can't find any documentation on our Jinja standards.

```suggestion
          {{ ( 'Appendix ' if is_appendix else '' ) + section.numbering + section.title }}
```